src/platforms/shared/setup/partners.setup.ts

```diff
@@ -0,0 +1,73 @@
+import {
+	AdSlotEvent,
+	BrandMetrics,
+	Captify,
+	communicationService,
+	DiProcess,
+	Experian,
+	Eyeota,
+	LiveConnect,
+	LiveRampPixel,
+	Lotame,
+	Nielsen,
+	PartnerPipeline,
+	utils,
+} from '@wikia/ad-engine';
+import { Injectable } from '@wikia/dependency-injection';
+
+@Injectable()
+export class PartnersSetup implements DiProcess {
```

Comment:
```suggestion
export class UcpPartnersSetup implements DiProcess {
```

This file is in "shared" directory, but this class won't be used on N&R platforms because these sites don't load all partners

Reply:
ok, thx